ui/pages/reservation-unit/[id].tsx

```diff
@@ -1049,140 +1043,105 @@ const ReservationUnit = ({
                     <p>
                       {reservationUnit.reservationsMaxDaysBefore > 0 &&
                         reservationUnit.reservationsMinDaysBefore > 0 && (
-                          <Trans i18nKey="reservationUnit:reservationInfo1-1">
-                            Voit tehdä varauksen{" "}
-                            <strong>
-                              aikaisintaan{" "}
-                              {{
-                                reservationsMaxDaysBefore: daysByMonths.find(
-                                  (n) =>
-                                    n.value ===
-                                    reservationUnit.reservationsMaxDaysBefore
-                                )?.label,
-                              }}
-                              {{
-                                unit: t(
-                                  `reservationUnit:reservationInfo1-${
-                                    reservationUnit.reservationsMaxDaysBefore ===
-                                    14
-                                      ? "weeks"
-                                      : "months"
-                                  }`
-                                ),
-                              }}
-                            </strong>{" "}
-                            ja{" "}
-                            <strong>
-                              viimeistään
-                              {{
-                                reservationsMinDaysBefore:
-                                  reservationUnit.reservationsMinDaysBefore,
-                              }}{" "}
-                              päivää etukäteen
-                            </strong>
-                            .
-                          </Trans>
+                          <Trans
+                            defaults="Voit tehdä varauksen <strong>aikaisintaan {{reservationsMaxDaysBefore}} {{unit}}</strong> ja <bold>viimeistään {{reservationsMinDaysBefore}} päivää etukäteen</bold>."
+                            values={{
+                              reservationsMaxDaysBefore: daysByMonths.find(
+                                (n) =>
+                                  n.value ===
+                                  reservationUnit.reservationsMaxDaysBefore
+                              )?.label,
+                              unit: t(
+                                `reservationUnit:reservationInfo1-${
+                                  reservationUnit.reservationsMaxDaysBefore ===
+                                  14
+                                    ? "weeks"
+                                    : "months"
+                                }`
+                              ),
+                              reservationsMinDaysBefore:
+                                reservationUnit.reservationsMinDaysBefore,
+                            }}
+                            components={{ bold: <strong /> }}
+                          />
```

Comment:
Missing i18nKey. Need to refactor the translation tags.
See https://react.i18next.com/latest/trans-component#alternative-usage-which-lists-the-components-v11.6.0